lib/system/threadlocalstorage.nim

```diff
@@ -144,12 +144,12 @@ else:
       tv_sec: Time
       tv_nsec: clong
 
-  proc pthread_attr_init(a1: var PthreadAttr) {.
+  proc pthread_attr_init(a1: var Pthread_attr) {.
     importc, header: pthreadh.}
-  proc pthread_attr_setstacksize(a1: var PthreadAttr, a2: int) {.
```

Comment:
Are these changes relevant to the fix?

Reply:
Well, they are relevant. I had compilation failures because of the new enforced style checks. But they are indeed from a different issue.